store/docs/soap-admin.txt

```diff
@@ -3168,3 +3168,43 @@ Corresponding API for outgoing/sent email filters (message structure same as abo
 </ModifyOutgoingFilterRulesRequest>
 
 <ModifyOutgoingFilterRulesResponse/>
+------------------------------
+ContactBackup API
+------------------------------
+<ContactBackupRequest [task="start|stop|status"]>
+    [<mailboxIds>
+        <id>xxx</id>
+        <id>xxx</id>
+        <id>xxx</id>
+        ...
+    </mailboxIds>
+    ]
+</ContactBackupRequest>
+
+task : It's an optional attribute. If not mentioned, "start" considered by default.
```

Comment:
why? this was discussed and decided while re-grooming.

Reply:
I was not part of the re-grooming and I don't see any of the decisions made while re-grooming in the ticket. So, please make sure whatever decisions are made, they should be mentioned in ticket to avoid scenarios like this.